### ISSUE TITLE
[NBug] An unexpected internal error occurred in the Win32 call: 80...

### ISSUE THREAD

**Issue by sxp200422** [gitextensions/gitextensions#9579]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.ComponentModel.Win32Exception: An unexpected internal error occurred in the Win32 call: 80004005 ---> System.Runtime.InteropServices.COMException: エラー HRESULT E_FAIL が COM コンポーネントの呼び出しから返されました。

   --- End of inner exception stack trace ---
   at Microsoft.WindowsAPICodePack.Dialogs.NativeTaskDialog.NativeShow()
   at Microsoft.WindowsAPICodePack.Dialogs.TaskDialog.ShowCore()
   at GitUI.NBugReports.BugReportInvoker.Report(Exception exception, Boolean isTerminating)
   at GitExtensions.Program.<>c.<Main>b__1_3(Object s, ThreadExceptionEventArgs e)
   at System.Windows.Forms.Application.ThreadContext.OnThreadException(Exception t)
   at System.Windows.Forms.Control.WndProcException(Exception e)
   at System.Windows.Forms.Control.ControlNativeWindow.OnThreadException(Exception e)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

```

## Additional information
Do nothing, just leave GE for night until now ,image below maybe useful to you.
![#9529-before](https://user-images.githubusercontent.com/12292750/133559878-696b23a9-1b24-4455-b876-a5126411bfdd.png)
![#9529-after](https://user-images.githubusercontent.com/12292750/133559883-c2dc6e7c-c833-4e64-808f-2f009dc794bb.png)

## Environment
- Git Extensions 3.5.3.12551
- Build 00604a59a673142a3d34a22aa657466a922977f3
- Git 2.32.0.windows.2
- Microsoft Windows NT 6.2.9200.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
Fix #9566 for this issue is under test.
Can you test the portable version generated by the build ( https://ci.appveyor.com/project/gitextensions/gitextensions/builds/40747903/artifacts ) and confirm it is working good for you?

The reason of the problem was that the GDI count increase (like you could see in your screenshot) and reach 10000 so GitExtension crash.

So, please, check that the number don't increase with time...

**Comment by gerhardol** [comment]
Duplicate of #9429
Related to theming.
